app/jobs/send_reminder_emails_job.rb

```diff
@@ -0,0 +1,13 @@
+class SendReminderEmailsJob < ApplicationJob
+  def perform(year)
+    reminders.each do |reminder|
```

Comment:
@d-a-v-e Do we want to batch these up into groups of 100/500/1000? Thinking 500 might be the sweet spot, if there are several thousand, but don't fully have a handle on numbers we are expecting. So 100 might be a safer batching number

Reply:
good shout, seemed like theres only going to be a few but no reason to not batch them.